### ISSUE TITLE
Dead Slack Invite Link

### ISSUE THREAD

**Issue by hayleycd** [sigstore/docs#319]
**Description**
In the quick start there is a dead slack invite at the top of the page (https://docs.sigstore.dev/signing/quickstart/). It should be updated.

**Comment by hayleycd** [comment]
The invite link is also dead here: https://docs.sigstore.dev/certificate_authority/overview/

**Comment by AndreVirtimo** [comment]
Is there an active invite link anywhere?

**Comment by unusualevent** [comment]
yeah, same. tried signing up after registering for SigStoreCon. @dlorenc the slack links on the website are down.
